src/collections/Breadcrumb/BreadcrumbSection.js

```diff
@@ -0,0 +1,98 @@
+import React, { PropTypes } from 'react'
+import cx from 'classnames'
+import META from '../../utils/Meta'
+import { customPropTypes, useKeyOnly } from '../../utils/propUtils'
+
+/**
+ * A section sub-component for Breadcrumb component.
+ *
+ * @see Breadcrumb
+ */
+function BreadcrumbSection(props) {
+  const {
+    active, children, className, link, href, onClick, ...rest,
+  } = props
+
+  /**
+   * Handles onClick event. If onClick is not defined, does nothing.
+   * */
+  const handleClick = (e) => {
+    if (onClick) onClick(e)
+  }
+
+  const classes = cx(
+    useKeyOnly(active, 'active'),
+    className,
+    'section',
+  )
+
+  // Handle link and onClick props.
```

Comment:
I think the code is clear enough here, let's remove the comment and examples for link/onClick, href, and the default variant.